client/components/marketing-survey/gsuite-cancel-purchase-dialog/index.jsx

```diff
@@ -235,11 +240,15 @@ GSuiteCancelPurchaseDialog.propTypes = {
 };
 
 export default connect(
-	( state, { purchase } ) => {
+	( state, { purchase, site, domain } ) => {
```

Comment:
As noted below, `domain` is being derived from `purchase.meta` on line 245.
```suggestion
	( state, { purchase, site } ) => {
```

Reply:
Right!